main.js

```diff
@@ -0,0 +1,240 @@
+    }
+    if( c == 2)
+    {
+        document.getElementById('div4').style.display = "block";
+        c=0;
+    }
+}
+
+
+
+document.getElementById("saveQuestion").onclick = function(e){
+    e.preventDefault();
+    addQuestion();
+
+}
+
+let survey = [];
+
+
+
+function addQuestion()
+{
+    let surveySet = {
+        sq : "",
+        options : []
+    }
+    surveySet.sq = document.getElementById('surveyQuestion').value;
+    surveySet.options = [
+                        document.getElementById('option1').value,                
+                        document.getElementById('option2').value,
+                        document.getElementById('option3').value,
+                        document.getElementById('option4').value,
+                        ]
+    if( (surveySet.sq.length == 0 ) )
+    {
+        alert("NO DATA");
+    }
+    else
+    {
+        survey.push(surveySet);
+        alert("Your Survey Question Saved!")
+        document.querySelector('form').reset();
+    }
+    // return surveySet;
+    document.getElementById('div3').style.display = "none";
+    document.getElementById('div4').style.display = "none";
+    document.getElementById('addMoreOptions').style.display = "block";
+
+    
+}
+
+// const ss = addQuestion();
+
+function displaySurveyArray()
+{
+    console.log('Length of Survey Array is : ',survey.length);
+    console.log('Items in the Survey Array ');
+    console.log(survey);
+    
+}
+
+// ------------------------------------------------------------------------------------------------------------------------//
+document.getElementById("finishButton").onclick = function(e){
+
+    alert("Saved all your questions and options. Click on ok button to see your survey form!")
+    e.preventDefault();
+    // console.log('-------------SURVEY ARRAY-------------');
+    // displaySurveyArray();
+
+       document.getElementById('finalSurveyForm').style.display = "block";  
+       document.getElementById('surveyBuildingDiv').style.display = "none";
+       document.getElementById('finishButton').style.display = "none";  
+       document.getElementById('welcomeUser').style.display = "none";
+
+       for(let i=0; i< survey.length ; i++)
+       {
+        const body = document.body;
+        const div = document.createElement("div");
+        div.className = "newClass";
+        body.append(div);
+
+
+        const strong = document.createElement("strong")
+        strong.innerText ="Q No:" + (i+1) + " - " + survey[i].sq;
+        div.append(strong);
+
+
+        for(let j=0; j<4 ; j++)
+        {
+            if(survey[i].options[j]!="")
+            {
+                const label = document.createElement("label")
+                const br = document.createElement("br")
+                div.append(br)
+                label.innerText = "option "+ (j+1)  +" : " +survey[i].options[j];
+                div.append(label) 
+            }               
+        }
+
+       }
+
+
+}
+
+
+
```

Comment:
Don't forgot you need to create different surveys, and survey set should be your final outcome. try to use my sdk layer. and implement the feature of managing surveys and qusetions---> user can delete or edit survey and also edit or delete qstns in a survey